misc/pytorch_toolkit/chest_xray_screening/tools/test.py

```diff
@@ -0,0 +1,135 @@
+import os
+import os.path
+import numpy as np
+import time
+import sys
+import csv
+import argparse
+import torch
+from torchvision import models
+import torch.nn as nn
+import torch.backends.cudnn as cudnn
+import torchvision
+import torchvision.transforms as transforms
+import torch.optim as optim
+import torch.nn.functional as tfunc
+from torch.utils.data import Dataset
+from torch.utils.data.dataset import random_split
+from torch.utils.data import DataLoader
+from torch.optim.lr_scheduler import ReduceLROnPlateau,StepLR
+from PIL import Image
+import torch.nn.functional as func
+import random
+from sklearn.metrics.ranking import roc_auc_score
+import sklearn.metrics as metrics
+import json
+from pathlib import Path
+from dataloader import RSNADataSet
+
+
+class RSNAInference():
+
+    #---- Computes area under ROC curve 
+    #---- dataGT - ground truth data
+    #---- dataPRED - predicted data
+    #---- classCount - number of classes
+    
+    def computeAUROC (dataGT, dataPRED, classCount):
```

Comment:
You define this function but you don't use it. On line 78 you use the same function defined in `RSNATrainer` class but you do not import this class. So it would be better to extract this function into separate file and import it from there to `Trainer` and `Test` classes.

Reply:
Created a separate function outside the class.